learning/dashboard/Group-5/OPSG5.md

```diff
@@ -0,0 +1,61 @@
+# How to download VS Code.
+
+## 1. Go to your prefered web browser and type download VS code and click on the first link. <br>
+![image](https://user-images.githubusercontent.com/123230201/227720415-691dea50-8d1f-4559-8eae-97150b4c3247.png)
+
+## 2. After Clicking on the first link click windows to download. <br>
+![image](https://user-images.githubusercontent.com/123230201/227720566-27441bf6-6575-4677-be4c-131458e23fd3.png)
+
+## 3. Wait for the download to start and finish.
+
+## 4. After the VS Code has finsihed downloading go through the setup process by clicking next and wait for it to download. 
+
+# How to install Angular Snippets extensions in VS Code 
+
+## After you finish downloading VS Code and setting it up, the next step is to download the extensions. In order to download the extensions in VS Code, we need to do the following steps. 
+
+## 1. Open VS Code and click on the extension button on the left side of VS Code. 
+![image](https://user-images.githubusercontent.com/123230201/227721833-f5057b9e-7113-435c-add3-8ffdc125c82d.png)
+
+## 2. Type Angular Snippets and click on the first option and click install and wait for it to install.
+![image](https://user-images.githubusercontent.com/123230201/227721915-a0e28d2c-f0ca-44a8-98cb-f3a7df7318ab.png) 
+#
+# How to install JSLint extension in VS Code.
+
+## After you've finished installing the Angular Snippets extension on VS code, the next step is to install the JSLint extension. In order to download the extension VS Code, we need to do the following steps.
+
+## 1. Open VS Code and click on the extension button of the left side of VS Code. 
+![image](https://user-images.githubusercontent.com/123230201/227721833-f5057b9e-7113-435c-add3-8ffdc125c82d.png)
+
+## 2. Type JSLint and click on first option and click install and wait for it to install. 
+![image](https://user-images.githubusercontent.com/123230201/227722349-5c408e0e-188e-40fc-8cd1-08530beeac9a.png) <p> 

```

Comment:
Consider adding some information about the JSLint extension.
What is the usage and how it helps users.